static/css/styles.css

```diff
@@ -0,0 +1,108 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
@TitovIvan 
 is it ok, to collect tags like this (huge amount of tags)?
 _need to refactor_

Reply:
this code used for different browsers to make them show content equal (as the same)